mesheryctl/internal/cli/root/system/logout.go

```diff
@@ -0,0 +1,43 @@
+package system
+
+import (
+	"io/ioutil"
+
+	"github.com/layer5io/meshery/mesheryctl/internal/cli/root/config"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+
+	log "github.com/sirupsen/logrus"
+)
+
+var logoutCmd = &cobra.Command{
+	Use:   "logout",
+	Short: "Remove authentication for Meshery Server",
+	Long: `
+Remove authentication for Meshery Server
+
+This command removes the authentication token from the user's filesystem`,
+	Args: cobra.MinimumNArgs(0),
+	RunE: func(cmd *cobra.Command, args []string) error {
+		mctlCfg, err := config.GetMesheryCtl(viper.GetViper())
+		if err != nil {
+			return errors.Wrap(err, "error processing config")
+		}
+
+		token, err := mctlCfg.GetTokenForContext(mctlCfg.CurrentContext)
+		if err != nil {
+			log.Error("failed to find token path for the current context")
+			return nil
+		}
+
+		// Replace the content of the token file with empty content
```

Comment:
@utkarsh-pro , we are only cleaning up the token on client(CLI), we need to invalidate the session on Meshery server as well, right? Are we relying on the session to timeout ?

Reply:
@hexxdump Meshery Server does not create any sessions. If token is present (and is valid) then the request will be authorized or else it will be rejected.